src/components/chat/chat.js

```diff
@@ -7,23 +7,26 @@ import 'codemirror/mode/javascript/javascript';
 import 'codemirror/lib/codemirror.css';
 import Overlay from './overlay';
 
+import ModalEndChat from './../modal/modal-end-chat';
+
 class Chat extends React.Component {
 
   textArea = React.createRef();
 
   state = {
     keepChangeEditor: '',
-    roomId: this.props.location.pathname.split('/chat/')[1],
+    roomId: this.props.location.pathname.split('/')[2],
+    // roomId: this.props.location.pathname.split('/chat/')[1], why is this duplicated?
```

Comment:
Thanks for explaining why a second one is created.. let's remove the comment for the duplicated one in the next update.